src/test/java/resources/batch.feature

```diff
@@ -94,6 +127,10 @@ Feature: Batch Details API test
   Scenario: BD-010 Batch summary with failure percent API Test
     Given I have a batch id from previous scenario
     And I have tenant as "gorilla"
+    And I have the demo csv file "ph-ee-bulk-demo-6.csv"
```

Comment:
put all test data files into a specific folder ( suggestion: testdata) instead of  root path

Reply:
File is already in resource folder. Sorry for the confusion I just checked.